seeders/pending-applications.js

```diff
@@ -0,0 +1,29 @@
+'use strict';
+const moment = require('moment');
+const uuidv4 = require('uuid/v4');
+const pendingApplications = require('./data/pending-applications-data.json');
+
+const prepareData = () => {
+  pendingApplications.data.forEach(application => {
+    application.id = uuidv4();
+    application.createdAt = moment().toDate();
+    application.updatedAt = moment().toDate();
+  });
+  pendingApplications.data[0].expiresAt = moment().add('30', 'days').toDate();
+  pendingApplications.data[1].expiresAt = moment().add('14', 'days').toDate();
+  pendingApplications.data[2].expiresAt = moment().add('2', 'days').toDate();
+  pendingApplications.data[3].expiresAt = moment().toDate();
+  pendingApplications.data[4].expiresAt = moment().subtract('5', 'days').toDate();
```

Comment:
dynamically change the dates based on the current.

Reply:
Id rather generate the whole thing using `faker` if we can (see form.test.js for an example) than use a static fixture

Reply:
Sure. Thought, the prop `expiresAt` will still need adding/subtracting 'x' days as this needs to be dynamic to the current date for various scenarios .ie. seeding data for 30, 14, 2 day expiries each.